core/src/main/java/bio/terra/pearl/core/service/kit/KitRequestService.java

```diff
@@ -307,6 +315,30 @@ public KitRequest assemble(
         return kitRequest;
     }
 
+    public KitRequest insertKitRequest(
+            UUID adminUserId,
+            Enrollee enrollee,
+            KitRequestDto kitRequestDto) {
+
+        KitType kt = kitTypeDao.findByName(kitRequestDto.getKitType().getName()).get();
+        KitRequest kitRequest = KitRequest.builder()
+                .creatingAdminUserId(adminUserId)
+                .enrolleeId(enrollee.getId())
+                .sentToAddress(kitRequestDto.getSentToAddress())
+                .status(kitRequestDto.getStatus())
+                .skipAddressValidation(kitRequestDto.isSkipAddressValidation())
+                .kitTypeId(kt.getId())
+                .createdAt(kitRequestDto.getCreatedAt() == null ? Instant.now() : kitRequestDto.getCreatedAt())
```

Comment:
setting the `createdAt` won't do anything here -- `create` auto-sets the createdAt to now.